### PR TITLE
TreeView: fix axe accessibility issues

### PR DESCRIPTION
**Pull request by Zedwag**

### CODE REVIEW

File: packages/devextreme/js/ui/hierarchical_collection/ui.hierarchical_collection_widget.js [hunk 1]
```diff
@@ -105,7 +105,17 @@ const HierarchicalCollectionWidget = CollectionWidget.inherit({
...
     _getIconContainer: function(itemData) {
-        return itemData.icon ? getImageContainer(itemData.icon) : undefined;
+        if(!itemData.icon) {
+            return undefined;
+        }
+
+        const $imageContainer = getImageContainer(itemData.icon);
+
+        if($imageContainer.is('img')) {
+            $imageContainer.attr('alt', itemData.text ?? `${this.NAME} item icon`);
```
Comment (marker-dao) [inline packages/devextreme/js/ui/hierarchical_collection/ui.hierarchical_collection_widget.js:115]: I think we should use messageLocalization.format() here for ```${this.NAME} item icon```.

Comment (Zedwag) [reply packages/devextreme/js/ui/hierarchical_collection/ui.hierarchical_collection_widget.js:115]: I think it's too much here

Comment (Zedwag) [reply packages/devextreme/js/ui/hierarchical_collection/ui.hierarchical_collection_widget.js:115]: it's supposed to be super rare case when we use img icon + do not use text

**Comment by EugeniyKiyashko** [comment]
LGTM

**Review by EugeniyKiyashko — APPROVED** [review]

**Review by marker-dao — APPROVED** [review]

**Pull request merged**